### PR TITLE
ci: Empty Azure Pipelines

### PR DESCRIPTION
**Pull request by Soap-141**
GitHub Issue: #

## Proposed Changes

 - [ ] Bug fix
 - [ ] Feature
 - [ ] Code style update (formatting)
 - [ ] Refactoring (no functional changes, no api changes)
 - [x] Build or CI related changes
 - [ ] Documentation content changes
 - [ ] Other, please describe:

## Description

Add empty azure pipelines.
When this merge I will be able to test my code in #2.

## Impact on version

- [ ] **Major**
  - The template structure was changed.
- [ ] **Minor**
  - New functionalities were added.
- [x] **Patch**
  - A bug in behavior was fixed.
  - Documentation was changed.

### Always applicable
No matter your changes, these checks always apply.
- [x] Your conventional commits are aligned with the **Impact on version** section.
- [x] Updated [CHANGELOG.md](../CHANGELOG.md).
  - Use the latest Major.Minor.X header if you do a **Patch** change.
  - Create a new Major.Minor.X header if you do a **Minor** or **Major** change.
  - If you create a new header, it aligns with the **Impact on version** section and matches what is generated in the build pipeline.
- [ ] ~~Documentation files were updated according with the changes.~~
- [ ] ~~Images about the template are referenced from the [wiki](https://github.com/nventive/UnoApplicationTemplate/wiki/Images) and added as images in this git.~~
- [ ] ~~TODO comments are hints for next steps for users of the template and not planned work.~~

### Contextual
Based on your changes these checks may not apply.
- [ ] ~~Automated tests for the changes have been added/updated.~~
- [ ] ~~Tested on all relevant platforms~~

### CODE REVIEW

**Review by takla21 — APPROVED** [review]

File: .azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+﻿# Use this to schedule a build at defined days and times
```
Comment (jeanplevesque) [inline .azure-pipelines.yml:1]: Would it be cleaner to put this in the build folder? So that everything pipeline related is in the same place?

Comment (Soap-141) [reply .azure-pipelines.yml:1]: I think it's best to have this file at the root of the project. Let me check if I can find anything.

Comment (Soap-141) [reply .azure-pipelines.yml:1]: I will also remove the dot. See https://learn.microsoft.com/en-us/azure/devops/pipelines/customize-pipeline?view=azure-devops#understand-the-azure-pipelinesyml-file for more details.

Comment (Soap-141) [reply .azure-pipelines.yml]: Done.

**Review by carlh98 — APPROVED** [review]

File: .azure-pipelines.yml (hunk 1 above)
Comment (jeanplevesque) [reply .azure-pipelines.yml]: I'd rather have it in the build folder. That way all the files associated to pipelines are in 1 place instead of 2.

**Pull request merged**